upsert.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rake', '~>10.1.1'
 
   if RUBY_VERSION >= '1.9'
-    gem.add_development_dependency 'activerecord-import'
+    gem.add_development_dependency 'activerecord-import', '0.3.1'
```

Comment:
I included a note, but I went through the versions until I found the most recent that worked (0.11.0).  Something changed between it and 0.12.0 that is causing it to fail.  Also, Postgres support for UPSERT was added in 0.12.0 so the benchmarks may be much closer in that case now.